src/chunk_list.h

```diff
@@ -855,20 +852,20 @@ static inline bool chunk_is_forin(chunk_t *pc)
 }
 
 
-void set_chunk_type_real(chunk_t *pc, c_token_t tt);
+void set_chunk_type_real(chunk_t *pc, c_token_t tt, const char *func, int line);
 
 
-void set_chunk_parent_real(chunk_t *pc, c_token_t tt);
+void set_chunk_parent_real(chunk_t *pc, c_token_t tt, const char *func, int line);
 
 
-#define set_chunk_type(pc, tt)      do { \
-      LOG_FUNC_CALL();                   \
-      set_chunk_type_real((pc), (tt));   \
+#define set_chunk_type(pc, tt)      do {                   \
+      LOG_FUNC_CALL();                                     \
```

Comment:
BTW, passing caller information to the actual functions is fine, but I wonder if that makes `LOG_FUNC_CALL` redundant?

Reply:
Yes, it is.
LOG_FUNC_CALL is only used in the files chunk_list.h, semicolons.cpp.